recipes/server.rb

```diff
@@ -28,8 +28,23 @@
 
 package [ 'avahi-daemon', 'avahi-utils' ] if platform_family?('debian')
 
+dpkg_package 'plexmediaserver' do
+  source cached_package_file
+  notifies :restart, 'service[plexmediaserver]'
+  options '--force-confold'
+  action :install
+  version node['plexapp']['version']
+end if platform_family?('debian')
+
 package 'plexmediaserver' do
   source cached_package_file
+  notifies :restart, 'service[plexmediaserver]'
+end unless platform_family?('debian')
```

Comment:
See https://github.com/jtimberman/plexapp-cookbook/issues/10, and https://github.com/chef/chef/issues/1763.